client/main.lua

```diff
@@ -191,17 +182,15 @@ local function hotwire(vehicle, plate)
             combat = true,
         }
     }) then
-        StopAnimTask(cache.ped, "anim@amb@clubhouse@tutorial@bkr_tut_ig3@", "machinic_loop_mechandplayer", 1.0)
         if (math.random() <= config.hotwireChance[GetVehicleClass(vehicle)]) then
             TriggerServerEvent('qb-vehiclekeys:server:AcquireVehicleKeys', plate)
         else
             TriggerServerEvent('hud:server:GainStress', math.random(1, 4))
-            exports.qbx_core:Notify(locale("notify.failed_lockedpick"), 'error')
+            exports.qbx_core:Notify(locale("notify.failed_keys"), 'error')
         end
         Wait(config.timeBetweenHotwires)
         isHotwiring = false
     else
-        StopAnimTask(cache.ped, "anim@amb@clubhouse@tutorial@bkr_tut_ig3@", "machinic_loop_mechandplayer", 1.0)
         isHotwiring = false
```

Comment:
nit; Since this is common in both cases, can move outside of the if condition altogether.

Reply:
right, I didn't notice that